vck-openid/src/commonMain/kotlin/at/asitplus/wallet/lib/oidvci/Json.kt

```diff
@@ -8,5 +9,6 @@ val jsonSerializer by lazy {
         encodeDefaults = false
         ignoreUnknownKeys = true
         isLenient = true
+        serializersModule = vckJsonSerializer.serializersModule
```

Comment:
Maybe now is a good time to rename the `jsonSerializer` into something more appropriate (or maybe even merge all into one?)

Reply:
I have opened an issue regarding this and will take care of this in #127